django/core/management/commands/migrate.py

```diff
@@ -20,8 +19,13 @@
 
 class Command(BaseCommand):
     help = "Updates database schema. Manages both apps with migrations and those without."
+    requires_system_checks = False
 
     def add_arguments(self, parser):
+        parser.add_argument(
+            '--skip-checks', action='store_true',
```

Comment:
This also seems to add a `--skip-checks` option to migrate (and below changes behaviour to only check the database that is being migrated, and I think also to run non-database-tagged checks). SHould this be documented in the commit message?

Reply:
`--skip-checks` is a default options that every command allows. This change just take it into account in skipping database checks.

Reply:
Can we move using the `skip-checks` option to a separate (first) commit?, also remove adding `--skip-checks` argument because it's already added by `BaseCommand.`

Reply:
OK we cannot move it to a separate commit because it's strictly related :disappointed: